client/simple/package.json

```diff
@@ -1,36 +1,51 @@
 {
   "$schema": "https://json.schemastore.org/package.json",
-  "name": "simple",
+  "name": "@searxng/theme-simple",
+  "version": "0.0.0",
   "private": true,
   "license": "AGPL-3.0",
   "type": "module",
   "scripts": {
     "build": "npm run build:icons && npm run build:vite",
```

Comment:
with my [comment from above](https://github.com/searxng/searxng/pull/5073/files#r2255949344) in mind: isn't it better we *lint* before build?

We could implement a *lint* here ...